app/src/main/java/com/x8bit/bitwarden/ui/platform/feature/settings/exportvault/ExportVaultViewModel.kt

```diff
@@ -116,9 +117,22 @@ class ExportVaultViewModel @Inject constructor(
             is ExportVaultAction.Internal.ReceiveVerifyOneTimePasscodeResult -> {
                 handleReceiveVerifyOneTimePasscodeResult(action)
             }
+
+            is ExportVaultAction.Internal.OtpCodeResult -> handleOtpCodeResult(action)
         }
     }
 
+    private fun handleOtpCodeResult(action: ExportVaultAction.Internal.OtpCodeResult) {
+        mutableStateFlow.update {
+            it.copy(dialogState = null)
+        }
+        val toastMessage = when (action.result) {
+            is RequestOtpResult.Error -> R.string.generic_error_message.asText()
```

Comment:
Do we want this generic message or the message in the error?

Reply:
good call, `result.message ?: default`